dashboard/d_graphs.py

```diff
@@ -0,0 +1,51 @@
+"""Script to create altair graphs for dashboard."""
+import pandas as pd
+import altair as alt
+
+
+def create_bubble_chart(df: pd.DataFrame) -> alt.Chart:
+    """Returns a bubble chart of sentiment for a source by topic."""
+    return alt.Chart(df).mark_circle().encode(
+        x=alt.X('source_name:N', title='Source'),
+        y=alt.Y('avg_polarity_score:Q',
+                title='Average Polarity Score'),
+        size=alt.Size('article_count:Q', title='Number of Articles'),
+        color=alt.Color('source_name:N', title='Source'),
+        tooltip=['source_name', 'avg_polarity_score', 'article_count']
+    ).properties(
+        width=800,
+        height=400
+    )
+
+
+def create_scatter_graph(df: pd.DataFrame) -> alt.Chart:
+    """Returns a scatter graph for title vs content score."""
+    scatter_chart = alt.Chart(df).mark_point(filled=True).encode(
+        x=alt.X('title_polarity_score:Q',
+                scale=alt.Scale(domain=[-1, 1]),
+                axis=alt.Axis(title='Title Polarity Score', grid=True)),
+        y=alt.Y('content_polarity_score:Q',
+                scale=alt.Scale(domain=[-1, 1]),
+                axis=alt.Axis(title='Content Polarity Score', grid=True)),
+        color=alt.Color('source_name:N', title='Source'),
+        tooltip=['article_title', 'title_polarity_score',
```

Comment:
you can use tool tip to rename the hints themselves.
e.g:
```
tooltip=[alt.Tooltip(field="article_title", title="Article title"),
              alt.Tooltip(field="title_polarity_score", title="Title polarity score"),
              alt.Tooltip(field="content_polarity_score", title="Content polarity score"),
              alt.Tooltip(field="source_name", title="News Source")]
```

Reply:
Ooo thanks I'll do this now!